src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final int COLOR_COUNT = 6;
+
+    public static String getRandomColor() {
```

Comment:
```suggestion

    public String getRandomColor() {
```

Reply:
java: non-static method getRandomColor() cannot be referenced from a static context